community/cypher/cypher-planner-3.4/src/main/scala/org/neo4j/cypher/internal/compiler/v3_4/ast/rewriters/InliningContext.scala

```diff
@@ -49,7 +49,7 @@ case class InliningContext(projections: Map[LogicalVariable, Expression] = Map.e
     copy(projections = resultProjections, seenVariables = seenVariables ++ newProjections.keys)
   }
 
-  def spoilVariable(variable: Variable): InliningContext =
+  def spoilVariable(variable: LogicalVariable): InliningContext =
```

Comment:
I know we didn't do here, but why `spoil`?

Reply:
I have no idea... possibly refers to special treatment by being omitted from `projections`?